shell/browser/api/electron_api_cookies.cc

```diff
@@ -128,7 +128,7 @@ bool MatchesCookie(const base::Value::Dict& filter,
       !MatchesDomain(*str, cookie.Domain()))
     return false;
   absl::optional<bool> secure_filter = filter.FindBool("secure");
-  if (secure_filter && *secure_filter == cookie.IsSecure())
+  if (secure_filter && *secure_filter != cookie.IsSecure())
     return false;
   absl::optional<bool> session_filter = filter.FindBool("session");
   if (session_filter && *session_filter != !cookie.IsPersistent())
```

Comment:
Maybe also switch this for simplicity

```suggestion
  if (session_filter && *session_filter == cookie.IsPersistent())
```